orbyter-base-sys/3.1/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:20.04
+WORKDIR /build
+ENV LC_ALL=C.UTF-8 \
+    LANG=C.UTF-8 \
+    TZ=America/Los_Angeles
+# The time zone avoids prompts during cmake install
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
+    echo $TZ > /etc/timezone && \
+    apt-get update -qq && \
+    # Most installs copied from orbyter-base-sys:2.0 \
+    apt-get install -qqy --no-install-recommends \
+        # Update to python 3.8 \
+        python3.8=3.8.5-1~20.04 \
+        python3-pip \
+        build-essential \
+        libssl-dev \
+        libffi-dev \
+        python3.8-dev \
+        vim \
+        emacs \
+        screen \
+        git \
+        wget \
+        curl \
+        cmake \
+        htop \
+        rsync \
+        nodejs \
```

Comment:
Just FYI, this line is redundant as nodejs is a dependency of npm

Reply:
Makes sense. I'm just gonna let it ride and let the dependency manager figure it out. I don't think there's a downside, right?